be/src/olap/utils.h

```diff
@@ -218,13 +218,10 @@ class Errno {
 };
 
 inline bool is_io_error(Status status) {
```

Comment:
You'd better move this method to common/status.h

Reply:
LGTM